src/stories/Library/status-message/status-message.scss

```diff
@@ -0,0 +1,60 @@
+.status-message {
+  background-color: $color__global-secondary;
+  padding: $s-md;
+  display: flex;
+  align-items: center;
+
+  @include media-query__x-small {
+    display: flex;
+    align-items: center;
+    padding: $s-md $s-xl $s-md $s-xl;
+  }
+
+  &__icon {
+    margin-right: $s-sm;
+    min-width: 40px;
+  }
+
+  &__description {
+    color: $color__text-primary-black;
+    @include typography($typo__body-placeholder);
+    padding: $s-md $s-sm;
+    max-width: 82%;
```

Comment:
is there a reason why the description can only be max 82% wide?

Reply:
To be honest I have done a copy/paste of the error-message.scss, so I don't really know why it is like that. 😅